### ISSUE TITLE
Usage issue

### ISSUE THREAD

**Issue by ghendric** [karenpayneoregon/MicrosoftConnectionDialog#1]
Hi, I have a question. I followed the instructions on how to use this but when I try to open the connection dialog, I get the error 
System.IO.FileNotFoundException
  HResult=0x80070002
  Message=Could not load file or assembly 'System.Data.OracleClient, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'. The system cannot find the file specified.
  Source=Microsoft.Data.ConnectionUI.Dialog
 
I tried adding a reference to System.Data.OracleClient to my project just for the heck of it and ran it again and got the same error when trying to open the connection dialog. My project is a .net 8 solution. Is there anything else specific I need to do? I got all the references set.

Thanks

**Comment by karenpayneoregon** [comment]
Did you try from C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8\System.Data.OracleClient.dll

You might also try closing/reopening the solution.

**Comment by ghendric** [comment]
Yes, I tried all of that. I added the reference to my project through Nuget, then I tried just adding the reference to that dll after removing it first through Nuget. I was looking at the code in the MicrosoftConnectionDialog project and didn't see any reference to System.DataOracleClient but it works just fine there. Could my issue be that my project is a .NET 8 core solution?

**Comment by karenpayneoregon** [comment]
Being that the project is .NET Framework 4.8, and your project NET 8 your code is not compatible.

**Comment by ghendric** [comment]
I didn't think so. Thanks
